docs/nodes/nodes-library/nodes/Google/Google Calendar/README.md

```diff
@@ -0,0 +1,42 @@
+---
+permalink: /nodes/n8n-nodes-base.googleCalendar
+---
+
+# Google Calendar
```

Comment:
Could you please make sure that this exists under ` docs/nodes/nodes-library/nodes/Google Calendar/README.md`? Please make that that moving this would not break the relative links in the document.